shell/helpers.sh

```
@@ -0,0 +1,113 @@
+        return 1
+    fi
+    _dotbins_helpers_run sync "$@"
+}
+
+
+dotbins_tool_path() {
+    if [[ $# -ne 1 ]]; then
+        _dotbins_helpers_log error "usage: dotbins_tool_path <tool>"
+        return 1
+    fi
+    local bin_dir="$(_dotbins_helpers_bin_dir)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)